Code/Framework/AzToolsFramework/AzToolsFramework/Logger/AzQtTraceLogger.h

```diff
@@ -0,0 +1,45 @@
+/*
+ * All or portions of this file Copyright (c) Amazon.com, Inc. or its affiliates or
+ * its licensors.
+ *
+ * For complete copyright and license terms please see the LICENSE at the root of this
+ * distribution (the "License"). All use of this software is governed by the License,
+ * or, if provided, by the license below or the license accompanying this file. Do not
+ * remove or modify any license notices. This file is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ */
+
+#pragma once
+
+#include <AzCore/Debug/TraceMessageBus.h>
+#include <AzFramework/Logging/LogFile.h>
+
+namespace AzToolsFramework
+{
+    //! Connects and disconnects TraceMessageBus and allows for logging for O3DE Tools Applications
+    class AzQtTraceLogger : public AZ::Debug::TraceMessageBus::Handler
```

Comment:
Please rename this to just `TraceLogger` as there's nothing Qt specific to this at all - this otherwise looks generally useful and easy to extend though, nice job!